api/catalog/api/serializers/media_serializers.py

```diff
@@ -471,8 +466,9 @@ class MediaSearchRequestSourceSerializer(serializers.Serializer):
         """
 
         _field_attrs = {
-            "help_text": make_comma_separated_help_text(
-                search_controller.get_sources(media_type).keys(), "data sources"
+            "help_text": (
+                "A comma separated list of data sources; for valid sources see "
+                f"https://api.openverse.engineering/v1/{media_path}/stats/."
```

Comment:
Is there a way go get the base URL programmatically here? If we ever change the version or the base URL?

Reply:
Would be nice to add something like "please use the source_name as a value"

Reply:
Using Django's `resolve` would do it, I believe (and have the correct links per environment, too) but when I last tried to implement that it caused circular imports :cry: